core/templates/pages/exploration-player-page/exploration-player-page.component.spec.ts

```diff
@@ -87,6 +90,12 @@ describe('Exploration player page', function() {
       }));
     spyOn(PageTitleService, 'setPageTitle').and.callThrough();
 
+    CommandExecutorService.getOuterFrameEvents =
+    jasmine.createSpy('outerFrameEvents spy');
+    CommandExecutorService.sendParentReadyState =
+    jasmine.createSpy('sendParentReadyState spy');
+    CommandExecutorService.sendStateToOuterFrame =
+    jasmine.createSpy('sendStateToOuterFrame spy');
```

Comment:
```suggestion
    CommandExecutorService.getOuterFrameEvents = jasmine.createSpy(
      'outerFrameEvents spy');
    CommandExecutorService.sendParentReadyState = jasmine.createSpy(
      'sendParentReadyState spy');
    CommandExecutorService.sendStateToOuterFrame = jasmine.createSpy(
      'sendStateToOuterFrame spy');
```

Please indent continuation by 2 characters. Also, line-wrap at the ``(``, instead. Ditto elsewhere.